### PR TITLE
Closes #457: change search placeholder text

### LINKED ISSUE #457: Fix search block placeholder text
Search block currently says "Search" as the placeholder text. Change to say "Search this site"

### PR DESCRIPTION
**Pull request by barrettbaffert**
## Description

Search block currently says "Search" as the placeholder text. Change to say "Search Site..." matching Quickstart 1.0

## How Has This Been Tested?

Local desktop (resized browser to check mobile)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by barrettbaffert** [comment]
Biggest question: should this have been done as a `hook_form_alter` or `hook_form_form_id_alter` instead? I tried the following function in the `az_barrio.theme` file, but had no luck having the change to actually appear locally (used dpm to ensure function was working and cleared cache many times). 

```
/**
 * Add placeholder text to the search forms.
 */
function az_barrio_form_search_block_form_alter(&$form,\Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
  $form['keys'][['#attributes']['placeholder']] = t('Search Site…');
}
```

**Review by smayden — COMMENTED** [review]
Can't comment on the code questions, but from a UI perspective I now see Search Site..." as the placeholder.

Is it possible to update to "Search this site" (sentence case and adding "this")?

**Comment by barrettbaffert** [comment]
> Is it possible to update to "Search this site" (sentence case and adding "this")?

Absolutely, that's easy. I only matched what we're doing in QS1.0, but I will adjust. Makes sense to me!

File: themes/custom/az_barrio/templates/forms/input--search.html.twig [hunk 1]
```diff
@@ -10,5 +10,4 @@
  * @see template_preprocess_input()
  */
 #}
-<input{{ attributes.addClass("form-control") }} />{{ children }}
-
+<input{{ attributes.setAttribute('placeholder', 'Search this site…').addClass("form-control") }} />{{ children }}
```
Comment (danahertzberg) [inline themes/custom/az_barrio/templates/forms/input--search.html.twig:13]:
```suggestion
<input{{ attributes.setAttribute('placeholder', 'Search this site').addClass("form-control") }} />{{ children }}
```

**Review by danahertzberg — APPROVED** [review]

**Review by tadean — DISMISSED** [review]

**Comment by barrettbaffert** [comment]
Latest update includes aria label matching placeholder text
image

**Review by joeparsons — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by smayden — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Review by danahertzberg — APPROVED** [review]

**Pull request merged**